Bio/SearchIO/BlastIO/blast_xml.py

```diff
@@ -188,26 +188,41 @@
 _RE_ID_DESC_PATTERN = re.compile(" +")
 
 
-def _extract_ids_and_descs(concat_str):
-    # Given a string space-separate string of IDs and descriptions,
-    # return a list of tuples, each tuple containing an ID and
-    # a description string (which may be empty)
+def _extract_ids_and_descs(raw_id, raw_desc):
+    # Given values of the `Hit_id` and `Hit_def` elements, return a tuple
```

Comment:
You might as well turn this informative multi-line comment into a docstring (tagged ```... (PRIVATE).``` at the end of the summary first line).